src/riak_kv_qry_compiler.erl

```diff
@@ -172,6 +171,19 @@ expand_query(?DDL{table = Table, local_key = LK, partition_key = PK},
             {ok, SubQueries2}
     end.
 
+%% Check if any columns in the partition key have the DESC keyword applied.
```

Comment:
The intro to this comment is no longer accurate.

Also the function name itself is no longer accurate, and is a bit tough to parse. What about `is_last_partition_column_descending`?